apps/condo/domains/user/utils/serverSchema/userHelpers.js

```diff
@@ -0,0 +1,53 @@
+const z = require('zod')
+
+const conf = require('@open-condo/config')
+
+
+function uniqueArray (schema) {
+    return z
+        .array(schema)
+        .refine((items) => new Set(items).size === items.length, {
+            message: 'All items must be unique, no duplicate values allowed',
+        })
+}
+
+const AllowedFields = z.enum(['phone', 'email'])
+
+const IdentificationUserFieldsSchema = z.object({
+    staff: uniqueArray(AllowedFields).min(1).max(2).optional(),
+    resident: uniqueArray(AllowedFields).min(1).max(2).optional(),
+})
+
+/**
+ *
```

Comment:
?

Reply:
I don't really understand what you mean)